### PR TITLE
Feat/vectorized calculations

### PR DESCRIPTION
**Pull request by mkolopanis**
uses numpy vectorization to perform some calculations.

A little help on the way to #5

### CODE REVIEW

**Comment by mkolopanis** [comment]
not sure I understand why the python 3.7 tests are failing on travis-ci.com but succeeding travis-ci.org (also on all my local tests)

**Comment by mkolopanis** [comment]
Here's a before and after comparison of some OVRO LWA calculation timings.
Largest increase comes from the 2D calculation, but I have noticed some good speed ups in the phasing UVW one too in some HERA calculations.

Before:
image


After:
image

File: py21cmsense/_utils.py [hunk 1]
```diff
@@ -71,7 +72,7 @@ def nonnegative(instance, att, x):
 
 def find_nearest(array, value):
     """Find closest value in `array` to `value`"""
-    idx = (np.abs(array - value)).argmin()
+    idx = np.abs(array.reshape(-1, 1) - value).argmin(0)
```
Comment (steven-murray) [inline py21cmsense/_utils.py:75]: Just return without setting idx

**Comment by steven-murray** [comment]
Oh, you should add yourself to the authors list and put the changes in the changelog...

**Pull request merged**